content/guides/autodiscovery.md

```diff
@@ -48,155 +86,169 @@ By default, the Datadog Agent includes Autodiscovery support for:
 - Redis
 - Riak
 
-These are provided by the configuration templates in the Datadog Agent `conf.d/auto_conf` directory.
+Storing templates as local files is easy to understand and doesn't require an external service. The downside is that you must redeploy the Agent container each time you change, add, or remove templates. You may also have to maintain your own docker-dd-agent container if you want to add your own templates.
```

Comment:
Should we mention the /conf.d volume they can expose, instead of rebuilding the image? See https://github.com/DataDog/docker-dd-agent/blob/master/entrypoint.sh#L172